helm-charts/support/values.jsonnet

```diff
@@ -1,4 +1,6 @@
 local cluster_name = std.extVar('2I2C_VARS.CLUSTER_NAME');
+local provider_name = std.extVar('2I2C_VARS.PROVIDER');
+local cost_monitoring_iam_role = std.extVar('2I2C_VARS.COST_MONITORING_IAM');
```

Comment:
so IAM role names are *deterministic*, with the account number the only variable part. so let's make the variable be the account number, and go from there?